platform/admin/ui/src/main/webapp/index.html

```diff
@@ -39,6 +39,7 @@
 
     </div>
     <div id="modalRegion"></div>
+    <div id="sessionTimeoutModalRegion"></div>
```

Comment:
I dont think this is being used anywhere - the modal appends to modalRegion on `wreqr.vent.trigger('showModal', new SessionTimeoutModal)`

Reply:
You're right. The region is there so we can add the timeout modal without affecting modals already being displayed, but I think that is a non-issue and we can remove this.

This brings up another point, though - do we handle stacked modals correctly? The timeout modal should always be on top, but I'm not sure that it is. We should verify this.

Reply:
Well right now for the admin modals only one modal will display at a time because the .show() 
 on this [line](https://github.com/brendan-hofmann/ddf/blob/8f564b13a45d5af651ed044ec3e2ab2b530c7e7d/platform/admin/ui/src/main/webapp/js/controllers/Modal.controller.js#L27) will automatically destroy the previous view.
Im not sure about the modals in intrigue though